lib/pub_sub/railties/active_record.rb

```diff
@@ -5,13 +5,25 @@ def self.included(klass)
         klass.send :include, InstanceMethods
         klass.send :extend, ClassMethods
 
-        klass.after_save :publish_changes, if: -> { changed? }
-        klass.after_destroy :publish_changes
-        klass.after_touch :publish_changes
+        # excplit cases that mark a record to be published
```

Comment:
Not sure what specs we have for this. But would be good to cover this behavior. Have you used this before? https://github.com/jdliss/shoulda-callback-matchers